README.md

```diff
@@ -179,18 +144,6 @@ PassageClient =
 devices = PassageClient.user.list_devices(user_id: user_id)
 ```
 
-The information available in the array of Passage Device struct returned by PassageClient.user.list_devices(user_id:):
-
-```ruby
-    Struct.new :id,
-               :cred_id,
-               :friendly_name,
-               :usage_count,
-               :updated_at,
-               :created_at,
-               :last_login_at,
-
-```
```

Comment:
Removing because these are now in the generated model docs. The source of truth must be the generated docs.